mulesoft/docs-site-ui
# DOCSINFRA-527 Updating UI to make result valid with CSP

Final version with multiple changes from marketing.  

Tested at https://docs.qax.mulesoft.com/general/
Test steps:

1. Inspect the page
2. Go to the console, verify no errors on load
3. Click about, no errors.
4. Go to the connectors landing page, verify that search works
5. Go to network and reload the page; verify no 4xx or 5xx errors
